packages/i18n/src/index.js

```diff
@@ -30,12 +20,11 @@ const DEFAULT_LOCALE_DATA = {
  */
 const logErrorOnce = memoize( console.error ); // eslint-disable-line no-console
 
+const i18n = new I18N();
```

Comment:
This default instance could also be exported. The default export looks like a good fit.